torchaudio/backend/soundfile_backend.py

```
@@ -82,9 +101,29 @@ def load_wav(filepath, **kwargs):
 
 @_mod_utils.requires_module('soundfile')
 @common._impl_save
-def save(filepath: str, src: Tensor, sample_rate: int, precision: int = 16, channels_first: bool = True) -> None:
+def save(
+        filepath: str,
+        src: Tensor,
+        sample_rate: int,
+        precision: int = 16,
+        channels_first: bool = True,
+        *,
+        compression: Optional[float] = None,
+) -> None:
     r"""See torchaudio.save"""
 
+    if precision:
+        warnings.warn(
+            '"precision" argument is depricated and removed in 0.8.0. In 0.8.0, '
+            'convert the input Tensor to dtype of desired precision.'
+        )
+
+    if compression is not None:
+        warnings.warn(
+            '"soundfile" backend\'s `save` function  does not support changing compression level. '
+            '`compression` argument is silently ignore.'
```

nit: silently ignored

---

will this parameter be removed?